api/numaresourcesoperator/v1/numaresourcesoperator_types.go

```diff
@@ -92,6 +86,12 @@ type NodeGroupConfig struct {
 	// +optional
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Topology info refresh period setting",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	InfoRefreshPeriod *metav1.Duration `json:"infoRefreshPeriod,omitempty"`
+	// Valid values are: true, false.
+	// Defaults to "false".
+	// +optional
+	// +kubebuilder:default=false
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="RTE Pause"
+	RTEPause *bool `json:"rtepause,omitempty"`
```

Comment:
you moved it into the right place, but I tend to prefer not using bools. Will add the k8s API design suggestions when I find it again

Reply:
will change it for the meanwhile to string (e.g enabled/disabled). just to understand better, what's not so good in bool? more complicated to understand compared to enabled/disabled maybe? bound us from adding a third option without large maintainance? or is there another reason?

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types - all the document is a great read which I keep always handy

Reply:
also https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md

Reply:
note that using bool is not per se automatically wrong. But is not a no-brainer either. We should just think about using them (vs. enums) and evaluate the consequences vs just slapping a bool here.

Reply:
thanks for the references, I'll look into them